Build/Chakra.Build.props

```diff
@@ -55,6 +55,10 @@
         %(PreprocessorDefinitions);
         INTL_ICU=1
       </PreprocessorDefinitions>
+      <PreprocessorDefinitions Condition="'$(ChakraICU)'=='static'">
+        %(PreprocessorDefinitions);
+        U_STATIC_IMPLEMENTATION=1
```

Comment:
>U_STATIC_IMPLEMENTATION=1 [](start = 8, length = 25)

Ah, I missed this. Thanks

Reply:
Not that I expect conflicts on this define name, but this exposes this define to every compilation unit (as opposed to ICU-related-only), no?

---
In reply to: [170405597](https://github.com/Microsoft/ChakraCore/pull/4737#discussion_r170405597) [](ancestors = 170405597)